client/src/app/site/interaction/components/action-bar/action-bar.component.ts

```diff
@@ -0,0 +1,120 @@
+import { ChangeDetectionStrategy, ChangeDetectorRef, Component } from '@angular/core';
+import { Router } from '@angular/router';
+
+import { marker as _ } from '@biesbjerg/ngx-translate-extract-marker';
+import { Observable } from 'rxjs';
+
+import { ComponentServiceCollector } from 'app/core/ui-services/component-service-collector';
+import { fadeInAnim, fadeInOutAnim } from 'app/shared/animations';
+import { BaseComponent } from 'app/site/base/components/base.component';
+import { ApplauseService } from '../../services/applause.service';
+import { CallRestrictionService } from '../../services/call-restriction.service';
+import { InteractionService } from '../../services/interaction.service';
+import { RtcService } from '../../services/rtc.service';
+
+const canEnterTooltip = _('Enter conference room');
+const cannotEnterTooltip = _('Add yourself to the current list of speakers to join the conference');
+@Component({
+    selector: 'os-action-bar',
+    templateUrl: './action-bar.component.html',
+    styleUrls: ['./action-bar.component.scss'],
+    changeDetection: ChangeDetectionStrategy.OnPush,
+    animations: [fadeInAnim, fadeInOutAnim]
+})
+export class ActionBarComponent extends BaseComponent {
+    public applauseLevel = 0;
+    public applauseDisabled = false;
```

Comment:
It appears you did not take the current OS3 for migrating this. These lines do not exist anymore.

https://github.com/OpenSlides/OpenSlides/blob/f1e2996c6ef880683a52b5bdc0548efa209fdd80/client/src/app/site/interaction/components/action-bar/action-bar.component.ts#L27-L30